backend-next/migrations/0000-initial.sql

```diff
@@ -0,0 +1,27 @@
+CREATE TABLE IF NOT EXISTS messages (
+  id uuid primary key default gen_random_uuid(),
+  created_at timestamp with time zone NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  username varchar(128) NOT NULL,
+  orig_msg text NOT NULL,
+  tl_msg text,
+  country varchar(2)
```

Comment:
I forget, is there a possibility where someone didn't have a country? Did we put antartica for those folks or something?

Reply:
We just left them blank lol